libraries/AR_WPNav/AR_WPNav.cpp

```diff
@@ -315,10 +323,17 @@ void AR_WPNav::update_pivot_active_flag()
         return;
     }
 
-    // if within 10 degrees of the target heading, exit pivot steering
-    if (yaw_error < AR_WPNAV_PIVOT_ANGLE_ACCURACY) {
+    uint32_t now = AP_HAL::millis();
+
+    // if within 10 degrees of the target heading, set start time of pivot steering
+    if (yaw_error < AR_WPNAV_PIVOT_ANGLE_ACCURACY && !_pivot_start_ms) {
```

Comment:
can we change this to (_pivot_start_ms > 0)?

Reply:
fixed